lib/middleware.js

```diff
@@ -309,6 +311,12 @@ var middlewares = module.exports = {
   },
   respondImages: function (req, res) {
     var data = Object.keys(images).map(function (id) {
+      // if we have full image info, we can return that instead
+      // assume if container_config exist on image, it is full object
+      if (images[id].container_config) {
+        return images[id];
+      }
+      // fake return object if we do not have all info
```

Comment:
repo tags set here 
https://github.com/Runnable/docker-mock/pull/39/files#diff-d7dfcb350720309421bbb465055aafdfR522

it should pick up repo-tags container_config should only exist if we are using real docker image format